partner_email_duplicate_warn/models/res_partner.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    same_email_partner_id = fields.Many2one(
```

Comment:
I was expecting a M2M fields. Specially if we install the module on an existing database. Don't think it's a big job to changes, just rename and remove a break in the compute function. Don't you think ?

Reply:
Maybe add a non stored field same_email_partner_names could do the job ? But links will not be possible.

Reply:
Yes it could work, although I think it would be a regression not to be able to click on the name.